README.md

```diff
@@ -0,0 +1,56 @@
+# Bitmark SDK for Java
+The official Bitmark SDK for Java
+
+[![Build Status](https://travis-ci.org/bitmark-inc/bitmark-sdk-java.svg?branch=master)](https://travis-ci.org/bitmark-inc/bitmark-sdk-java)
+
+## Setting Up
+
+### Prerequisites
+
+- Java version 1.8+
+
+### Installing
+
+#### Gradle
+```sh
+dependencies {
+    implementation 'com.bitmark.sdk:java-sdk:2.1.0'
+}
+```
+
+#### Maven
+```sh
+<dependency>
+    <groupId>com.bitmark.sdk</groupId>
+    <artifactId>java-sdk</artifactId>
+    <version>2.1.0</version>
+</dependency>
+```
+
+## Documentation
+
+Please refer to our [SDK Document](https://sdk-docs.bitmark.com/).
+
+
+## Sample code
+This is a [sample project](sample/). It shows how to use Bitmark SDK for Java.
+
+## Opening Issues
+If you encounter a bug with the Bitmark SDK for Java we would like to hear about it. Search the existing issues and try to make sure your problem doesn’t already exist before opening a new issue. It’s helpful if you include the version of the SDK, Java and OS you’re using. Please include a stack trace and reduced repro case when appropriate, too.
+
+
+## License
```

Comment:
I think we should put LICENSE in another md file and reference from here. We use one LICENSE for project and it should be referenced from each module's README file.